dangerfile.ts

```diff
@@ -1,10 +1,40 @@
-import { danger, warn } from 'danger'
+import { danger, fail, warn } from 'danger'
+import * as fs from 'fs'
+import * as path from 'path'
+
+const CHANGELOG_FILE = 'CHANGELOG.md'
+
+/**
+ * This function asserts that added entries into the changelog file are placed in the right section.
+ */
+const hasAddedLinesAfterVersionInChangelog = async (): Promise<boolean> => {
+  const changelogContent = fs.readFileSync(path.resolve(__dirname, CHANGELOG_FILE)).toString()
+  const versionLineNumber =
+    changelogContent
+      .split('\n')
+      .findIndex(line => line.startsWith('<!--------------------------------[ ')) - 1
```

Comment:
This seems to be too fragile. Adding or removing a single `-` will break if without being ever noticed.
I would rather search for the second `<!--`.

Reply:
Updated 👍